doc/developers/bug_triaging.rst

```diff
@@ -0,0 +1,146 @@
+.. _bug_triaging:
+
+Bug triaging and issue curation
+================================
+
+The `issue tracker <https://github.com/scikit-learn/scikit-learn/issues>`_
+is important to the communication in the project: it helps
+developers identify major projects to work on, as well as to discuss
+priorities. For this reason, it is important to curate it, adding labels
+to issues and closing issues that are not necessary.
+
+Working on issues to improve them
+--------------------------------------
+
+Improving issues increases their chances of being successfully resolved.
+Guidelines on submitting good issues can be found :ref:`here
+<filing_bugs>`. A third party can give useful feedback or even add
+comments on the issue, while core-developpers or members of the triage
+team can edit the issue description and title.
+
+The following actions are typically useful:
+
+  - documenting issues that are missing elements to reproduce the problem
+    such as code samples
+
+  - correcting incorrect use of code formatting
+
+  - making sure that the title and description are explicit about the
+    problem to be solved
+
+  - linking to related issues or discussions while briefly describing how
+    they are related, for instance "See also #xyz for a similar attempt
+    at this" or "See also #xyz where the same thing happened in
+    SomeEstimator" provides context and helps the discussion.
+
+.. topic:: Fruitful discussions
+
+   Online discussions may be harder than it seems at first glance, in
+   particular given that a person new to open-source may have a very
+   different understanding of the process than a seasonned maintainer.
+
+   Overall, it is useful to stay positive and assume good will. `The
+   following article
+   <http://gael-varoquaux.info/programming/technical-discussions-are-hard-a-few-tips.html>`_
+   explores how to lead online discussions in the context of open source.
+
+Triaging operations for members of the core and triage teams
+-------------------------------------------------------------
+
+In addition to the above, members of the core team and the triage team
+can do the following important tasks:
+
+- Update labels for issues and PRs
+
+- Follow up on stalled PRs, to see if they must be relabeled as
+  stalled and needing help (this is typically very important in the context
+  of sprints, where the risk is to create many unfinished PRs)
```

Comment:
I think that a link to the [documentation about stalled PRs](https://scikit-learn.org/stable/developers/contributing.html?highlight=stalled#stalled-pull-requests) would be appropriate here.
More generally, perhaps, a Pull Request workflow would be useful, together with the issue workflow below, but it can be addressed in another PR.